config.yaml

```diff
@@ -21,6 +21,7 @@ default:
    --plugin-option=tmpfs:required_ram_mb=39800"
```

Comment:
minor: In commit longer msg, imho "This patch adds an option" is clearer than "There's an option"